Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1078,6 +1078,29 @@ - (NSDictionary *)sanitizeQuotesFromInput:(NSDictionary *)input {
     return retDict;
 }
 
+- (void)processListOfApps {
+    dispatch_async(dispatch_queue_create("app_lister", NULL), ^{
+        BNCServerRequest *req = [[BNCServerRequest alloc] init];
+        req.tag = REQ_TAG_UPLOAD_LIST_OF_APPS;
+        NSMutableDictionary *post = [[NSMutableDictionary alloc] init];
```

Comment:
This dictionary doesn't need to be mutable, since you are adding a fixed set of items. Could be done w/ the shorthand as mentioned above.